File: SocialAppFlutter/social_app/lib/services/events/create_new_location_request.dart
class CreateNewLocationRequest {
  final double latitude;
  final double longitude;
  final String city;
  final String locationName;

  CreateNewLocationRequest(this.latitude, this.longitude, this.city, this.locationName);

  Map<String, dynamic> toJson() => 
  {
    'latitude': latitude,
    'longitude': longitude,
    'city': city,
    'locationName': locationName
  };
}

File: APIs/EventsService/EventsService/DTOs/Request/CreateNewLocationRequest.cs
namespace EventsService.DTOs.Request
{
    public class CreateNewLocationRequest
    {
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string City { get; set; } = string.Empty;
        public string LocationName { get; set; } = string.Empty;
        public int SportCategoryId { get; set; }
    }
}
